src/services/enum-utilities.ts

```diff
@@ -220,6 +220,11 @@ export default class EnumUtilities {
     return (item.name === FilingTypes.INCORPORATION_APPLICATION)
   }
 
+  /** Returns True if filing is a Notice of Withdraw. */
```

Comment:
Typo.

My fingers can't type "withdrawal" in one go, either. I always type "width..." 😆

Reply:
Thanks! updated